docs/sources/mimir/references/architecture/components/ruler/index.md

```diff
@@ -12,19 +12,19 @@ weight: 130
 The ruler is an optional component that evaluates PromQL expressions defined in recording and alerting rules.
 Each tenant has a set of recording and alerting rules and can group those rules into namespaces.
 
+Evaluating rules generates new samples. Those samples are then passed to an in-process [distributor](../distributor) to be ingested and made available for further queries.
+Configuration of the built-in distributor uses [its respective configuration parameters](../../../../configure/configuration-parameters/#distributor).
```

Comment:
[Nit] I think respective is for referring to multiple components, not necessary when just referring to one.
```suggestion
Configuration of the built-in distributor uses [its configuration parameters](../../../../configure/configuration-parameters/#distributor).
```